packages/app-project/src/screens/ProjectAboutPage/components/AboutNavLink.js

```diff
@@ -23,12 +23,12 @@ const AboutNavLink = ({ router, link }) => {
 
   return (
     <Box
-      background={isCurrentPage ? 'accent-1' : 'neutral-6'}
+      background={isCurrentPage ? 'accent-1' : { light: 'neutral-6' }}
```

Comment:
Similar comment here about the color object.

```suggestion
      background={isCurrentPage ? 'accent-1' : { light: 'neutral-6', dark: '' }}
```